packages/cdk/lib/stacks/engines/nextflow-engine-construct.ts

```diff
@@ -58,7 +58,6 @@ export class NextflowEngineConstruct extends EngineConstruct {
     engineLogGroup.grant(adapterRole, "logs:StartQuery");
 
     const lambda = this.renderAdapterLambda({
-      vpc: props.vpc,
```

Comment:
Won't the lambda need the `vpc` reference when it is being used with a custom VPC?

Reply:
Well this is the tricky thing. My "public mode" will not work if the lambdas need to be in a VPC: if it's in a public subnet then lambdas don't work, and if a private subnet then we're paying for a NAT gateway so my changes are pointless, and an isolated subnet would mean no access to AWS services. Therefore the lambda has to be in no VPC for my purposes.

Perhaps I could implement a check that says that the public subnet mode is mutually exclusive with the custom VPC? Would that be sufficient?

Reply:
I have made this change in my recent commits.